### DISCUSSION TITLE
next的getStaticProps中返回的图标显示不出来

### DISCUSSION THREAD

**Discussion by pangxzi** [unocss/unocss#2367] [Q&A]
<img width="1460" alt="image" src="https://user-images.githubusercontent.com/52907799/226093122-7edfc6f4-d104-4a06-8523-aab5e71306a6.png">

如果没有把图标的名字写在getStaticProps就可以显示

<img width="1500" alt="image" src="https://user-images.githubusercontent.com/52907799/226093198-b140c837-d5ba-4c10-afab-cb1b60f382ff.png">

**Comment by zyyv** [comment]
We temporarily close this due to the lack of enough information.
Please provide a [minimal reproduction](https://stackoverflow.com/help/minimal-reproducible-example) to reopen the issue.
Thanks.

> [Why reproduction is required](https://antfu.me/posts/why-reproductions-are-required)
